include/groonga/plugin.h

```diff
@@ -152,6 +152,16 @@ grn_plugin_logtrace(grn_ctx *ctx, grn_log_level level);
 
 typedef struct _grn_plugin_mutex grn_plugin_mutex;
 
+/**
+ * \brief Create a new object of \ref grn_plugin_mutex.
+ *
+ * Memory for the new object is obtained with \ref GRN_PLUGIN_MALLOC.
+ *
+ * \param ctx The context object.
+ *
+ * \return A newly created \ref grn_plugin_mutex object on success, `NULL` if
+ *         sufficient memory is not available.
```

Comment:
How about add such as the following description?
"THe new created grn_plugin_mutex object should be freed after use by calling `grn_plugin_mutex_close()`".

Reply:
https://github.com/groonga/groonga/blob/d3e0120790de90441afd33da63e7187ab0c06b26/include/groonga/cache.h#L83-L84

Added with the same wording as existing.